tests/openvino/utils_tests.py

```diff
@@ -101,6 +101,7 @@
     "llama4": "katuni4ka/tiny-random-llama-4-8E",
     "llava": "katuni4ka/tiny-random-llava",
     "llava_next": "katuni4ka/tiny-random-llava-next",
+    "llava_next_mistral": "/home/ilyas/optimum-intel/tiny-llava-next-mistral",
```

Comment:
It suppose to be a huggingface repo instead of local path, you may consider upload this `tiny-llava-next-mistral` to your hf repo. Thanks.